queue_services/entity-filer/src/entity_filer/filing_processors/correction.py

```diff
@@ -47,9 +47,7 @@ def process(correction_filing: Filing, filing: Dict, filing_meta: FilingMeta, bu
     )
 
     corrected_filing_type = filing['correction']['correctedFilingType']
-    # added CP, change of directors / change of address for CP is allowed
-    if business.legal_type in ['SP', 'GP', 'BC', 'BEN', 'CC', 'ULC', 'CP'] and \
```

Comment:
unnecessary if condition, this contains all legal types

Reply:
What if we add new legal types?

Reply:
it should be handled in allowable actions